CONTRIBUTING.md

```diff
@@ -56,33 +49,21 @@ send us a Pull Request. Following these steps will help us to include your code:
      ```
   3. Create your patch,  **including appropriate test cases**
   4. Follow our [Coding Style Guidelines](#style)
-  5. Run the AngularHint protractor testing suite using the `gulp` command, or run an AngularHint
-    module's unit tests using `karma start`. Ensure that all tests pass.
+  5. Run the AngularHint protractor testing suite using `yarn test-e2e`, or run an AngularHint
+    module's unit tests using `yarn test-unit`. Ensure that all tests pass.
   6. Commit your changes using a descriptive commit message that follows our [commit guidelines](#commit)
 
      ```shell
      git commit -a
      ```
     Note: the optional commit `-a` command line option will automatically "add" and "rm" edited files.
-  7. Build your changes locally to ensure all the tests pass
-    For AngularHint:
-
-    ```shell
-    gulp
```

Comment:
I haven't changed anything in that regard. But this step is run after you have commited your changes so it is not about including built files in the commit. This step just seemed a duplicate of step 5 (running tests), with extra info about modules, which do not exist any more.

(Also note that the default `gulp` task runs the tests.)